code/README.md

```diff
@@ -20,6 +20,20 @@ go install cogentcore.org/cogent/code/cmd/cogentcode@latest
 
 * See Releases on this github page for pre-built OS-specific app packages that install the compiled binaries.
 
+## MacOS environment variables
+
+When `Code` is run from an installed mac app bundle, it does not inherit the normal shell environment. To set environment variables in a way that extends to GUI apps, you must use:
+```sh
+launchctl setenv VAR VALUE
+```
+The `goal` shell automatically does this for you when setting any environment variables.
```

Comment:
When the user sets an environment variable in the interactive shell, there is absolutely no way that they expect it to make changes to the underlying system environment. At the very least this should only apply to things in the `.goal` file, given that I could do `set A test` or something in the interactive shell while experimenting, and I absolutely would not want that stored in the system forever!

Reply:
Moreover, designing a system that is wholly incompatible with most people's shells is not tenable. Cogent Code must support someone who uses bash and wants to be able to run commands.

Reply:
What about just running all commands through separate Goal processes in Cogent Code, which would bypass the environment variable issue entirely since Goal would get the right environment variables on startup? We will be replacing a lot of the current command infrastructure with a Goal terminal interface soon anyway.